src/compiler/crystal/tools/doc/html/type.html

```diff
@@ -17,9 +17,9 @@
 <div class="main-content">
 <h1 class="type-name">
 <% if type.program? %>
-  <%= type.full_name %>
+  <%= type.full_name.gsub(/::/, "::<wbr>") %>
```

Comment:
There's no need for a regular expression here

```suggestion
  <%= type.full_name.gsub("::", "::<wbr>") %>
```

Ditto below.